Adding adopter section and link to adopters.md. (#112)
project-akri/akri-docs: docs/community/contributing.md

For more details and the exact DCO text, you can visit [Developer Certificate of Origin](developercertificate.org).

## Adopters

> If you are leveraging Akri for your solution, we highly encourage you to add your organization or project information to our [adopter list](https://github.com/project-akri/akri/blob/main/ADOPTERS.md). Please submit a pull request against the [ADOPTERS.md in the Akri GitHub](https://github.com/project-akri/akri/blob/main/ADOPTERS.md). Your feedback is highly appreciated and would help us to justify the future investment in this project.

Reviewer: As worded, it seems to suggest that we are hesitant to invest in the project. Can we slightly modify it to:
```suggestion
If you are leveraging Akri for your solution, we highly encourage you to add your organization or project information to our [adopter list](https://github.com/project-akri/akri/blob/main/ADOPTERS.md). Please submit a pull request against the [ADOPTERS.md in the Akri GitHub](https://github.com/project-akri/akri/blob/main/ADOPTERS.md). Your participation helps us prioritize feature development in the project.
```
